app/scripts/views/components/barChart/barChart.js

```diff
@@ -723,13 +731,13 @@
                     Number(cbio.util.toPrecision(Number(_tmpValue), 3, 0.1));
                 }
 
-                // If the current tmpValue already bigger than maxmium number, the
+                // If the current tmpValue already bigger than maximum number, the
                 // function should decrease the number of bars and also reset the
-                // Mappped empty value.
+                // Mapped empty value.
                 if (_tmpValue >= max) {
```

Comment:
Could we just change the if condition to `_tmpValue < max` and remove `break` ?

Reply:
We need to break the for loop if the current value is higher than the max